x-pack/test/security_solution_api_integration/config/services/security_solution_ess_utils.ts

```diff
@@ -5,18 +5,30 @@
  * 2.0.
  */
 
+import { format as formatUrl } from 'url';
+import supertest from 'supertest';
 import { FtrProviderContext } from '../../ftr_provider_context';
 import { SecuritySolutionUtils } from './types';
 
 export function SecuritySolutionESSUtils({
   getService,
 }: FtrProviderContext): SecuritySolutionUtils {
   const config = getService('config');
-  const supertest = getService('supertest');
+  const supertestService = getService('supertest');
 
   return {
     getUsername: (_role?: string) =>
       Promise.resolve(config.get('servers.kibana.username') as string),
-    createSuperTest: (_role?: string) => Promise.resolve(supertest),
+    createSuperTest: async (role?: string, password: string = 'changeme') => {
+      if (!role) {
+        return supertestService;
+      }
+      const kbnUrl = formatUrl({
+        ...config.get('servers.kibana'),
+        auth: false,
+      });
+
+      return supertest.agent(kbnUrl).auth(role, password);
+    },
```

Comment:
```suggestion
    createSuperTest: async (role?: string, password: string = 'changeme') => {
      if (!role) {
        return supertestWithoutAuth;
      }
      
      return Promise.resolve(supertestWithoutAuth).auth(role, password);
    },
```

Reply:
`supertestWithoutAuth` is an instance of `supertest()` while what I'm returning here is an instance of `supertest.agent()` which has `.auth` method available. `supertest().auth` is not a function.

Reply:
👍 Thanks for the clarification!